tools/spelling/spelling_dictionary.txt

```diff
@@ -617,6 +617,7 @@ getaddrinfo
 getaffinity
 gethostname
 getifaddrs
+getname
```

Comment:
it looks like this is not needed; maybe it was in an intermediate state of this PR?

If I'm wrong go ahead an put it back.

Reply:
It's complaining for the comment I wrote which uses [`pthread_getname_np`](https://github.com/envoyproxy/envoy/pull/12011/files#diff-0cdb4fc7ec31ab3d8d7bfeecacaf3d60R244)

Reply:
pretty sure you can wrap the word in back ticks (maybe double back ticks?) to avoid the spell checker, i think thats the preferred way to handle these special words